CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 6.19.0
+
+This release makes a change to the data normalization process, limiting the number of entries or properties which will be included in any given array or object to 1000. Previously there was no limit, so in rare cases you may notice a change in your context data. If this is a problem, you can increase the limit with the new `maxNormalizationBreadth` setting. See [#4689](https://github.com/getsentry/sentry-javascript/pull/4689) for details.
```

Comment:
Just a thought: should we link to the docs page which explains the normalization settings? The problem in this case would be though that users could click on the link before the new setting is published in the docs.

Reply:
> The problem in this case would be though that users could click on the link before the new setting is published in the docs.

Yup, exactly. That's why I linked to the PR, because the PR has a link to the docs PR, which is as good as a link to the docs.